src/main/java/workshops/mock/AccountMockGenerator.java

```diff
@@ -10,82 +10,81 @@
-                new Account(AccountType.LO1, "1235478", new BigDecimal("1"), Currency.PLN),
-                //14
-                new Account(AccountType.RO1, "72446", new BigDecimal("0.01"), Currency.CHF),
-                //15
-                new Account(AccountType.RO1, "90753", new BigDecimal("0"), Currency.CHF),
-                //16
-                new Account(AccountType.RO2, "865423", new BigDecimal("0"), Currency.CHF),
-                //17
-                new Account(AccountType.LO2, "9612541", new BigDecimal("23500.86"), Currency.USD),
-                //18
-                new Account(AccountType.RO1, "971561", new BigDecimal("9999"), Currency.USD),
-                //19
-                new Account(AccountType.LO2, "97156221", new BigDecimal("10.00"), Currency.CHF),
-                //20
-                new Account(AccountType.RO1, "867151", new BigDecimal("109823.00"), Currency.PLN),
-                //21
-                new Account(AccountType.RO1, "862252", new BigDecimal("123771"), Currency.CHF),
-                //22
-                new Account(AccountType.RO2, "872562", new BigDecimal("1234"), Currency.PLN),
-                //23
-                new Account(AccountType.LO1, "86252", new BigDecimal("7332"), Currency.PLN),
-                //24
-                new Account(AccountType.RO2, "34563", new BigDecimal("2346"), Currency.PLN),
-                //25
-                new Account(AccountType.RO2, "1122", new BigDecimal("100"), Currency.USD),
-                //26
-                new Account(AccountType.RO1, "62222", new BigDecimal("1009"), Currency.PLN),
-                //27
-                new Account(AccountType.RO2, "73344", new BigDecimal("0"), Currency.PLN),
-                //28
-                new Account(AccountType.LO1, "8723212", new BigDecimal("2435"), Currency.USD),
-                //29
-                new Account(AccountType.RO2, "3457117", new BigDecimal("10000984"), Currency.PLN),
-                //30
-                new Account(AccountType.ROR2, "45218", new BigDecimal("108987.0"), Currency.CHF),
-                //31
-                new Account(AccountType.LO1, "24578", new BigDecimal("13873"), Currency.CHF),
-                //32
-                new Account(AccountType.LO1, "0000064", new BigDecimal("9766"), Currency.USD),
-                //33
-                new Account(AccountType.ROR2, "2322255", new BigDecimal("1000"), Currency.CHF),
-                //34
-                new Account(AccountType.ROR2, "666622", new BigDecimal("287"), Currency.CHF),
-                //35
-                new Account(AccountType.ROR2, "998292", new BigDecimal("1467"), Currency.CHF),
-                //36
-                new Account(AccountType.LO1, "938383", new BigDecimal("1600"), Currency.EUR),
-                //37
-                new Account(AccountType.LO1, "2018543", new BigDecimal("23000.86"), Currency.PLN)
-        );
-    }
-}
+	public List<Account> generate() {
+		return Arrays.asList(
```

Comment:
Dlaczego tu wszedzie zbedny tabulator przed new Account?

Reply:
akurat tego nie sprawdzałam, ale w CompanyMock tez tak mam więc prawdopodobnie po prostu tak był projekt już

Reply:
No to popraw, nie wazne jaki on byl, robisz PR, wiec sie podpisujesz pod tym, jest to pokazane jako twoja zmiana, wiec nie wiem kto to robil, ale git uwaza ze ty

Reply:
nie wiem czemu ale jak usuwam tab to jest okej a gdy tylko robię commita to on mi automatycznie wraca na to samo miejsce, w ustawieniach tez nie widzę problemu a mimo to to się automatycznie przesuwa ale tylko po commicie bo inaczej zostaje w miejscu.. wina gita?

Reply:
ok poprawione ale od ostatniego commita bo musiałam zmienić trochę ustawienia żeby przestało mi przeskakiwać

Reply:
Okej dobrze